arch/xtensa/src/esp32/esp32_touch.c

```diff
@@ -51,7 +51,7 @@
 #define TOUCH_PAD_FILTER_FACTOR_DEFAULT (4) /* IIR filter coefficient */
 #define TOUCH_PAD_SHIFT_DEFAULT         (4) /* Increase computing accuracy */
 #define TOUCH_PAD_SHIFT_ROUND_DEFAULT   (8) /* ROUND = 2^(n-1) */
-#define TOUCH_GET_IO_NUM(channel) (touch_channel_to_gpio[channel])
+#define TOUCH_GET_RTCIO_NUM(channel) (touch_channel_to_rtcio[channel])
```

Comment:
```suggestion
#define TOUCH_GET_RTCIO_NUM(channel)    (touch_channel_to_rtcio[channel])
```
nit: align to previous line